components/browser/Column.js

```diff
@@ -589,7 +589,7 @@ export default function Column(props) {
       const { filteredRows, queryIsInvalid } = filterCollections(
         colItems.map((p) => {
           const customLoad = getQuota(p)
-          const quotaNotReached = !customLoad || p.traverseEdgesCount < customLoad
+          const quotaNotReached = customLoad === undefined || p.traverseEdgesCount < customLoad
```

Comment:
If there is no custom load I think we don’t have to check this so I would remove the first part of the or

Reply:
I know, there are some reviewers that appear in the column that are not members of the reviewers group so they don't have a custom max papers edge so I think we shouldn't show them as available.